Cargo.toml

```diff
@@ -84,6 +84,7 @@ sysinfo = "0.29.10"
 
 [target.'cfg(target_os = "linux")'.dependencies]
 tun = { version = "0.7.5", features = ["async"] }
+sysinfo = "0.33.0"
```

Comment:
What's up with the sysinfo version for macOS? Can we update that in a seperate PR?

Reply:
yes, did see that and thought to update it in a separate PR